ts/features/fims/history/screens/HistoryScreen.tsx

```diff
@@ -1,37 +1,79 @@
-import { VSpacer } from "@pagopa/io-app-design-system";
-import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Body, Divider, IOStyles, VSpacer } from "@pagopa/io-app-design-system";
 import * as React from "react";
-import { SafeAreaView, Text, View } from "react-native";
-import LoadingScreenContent from "../../../../components/screens/LoadingScreenContent";
+import { FlatList, SafeAreaView, View } from "react-native";
+import { FooterActions } from "../../../../components/ui/FooterActions";
 import { useHeaderSecondLevel } from "../../../../hooks/useHeaderSecondLevel";
+import I18n from "../../../../i18n";
 import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { FimsHistoryListItem } from "../components/ListItem";
+import { LoadingFimsHistoryItemsFooter } from "../components/Loaders";
 import { fimsHistoryGet } from "../store/actions";
-import { fimsHistoryPotSelector } from "../store/selectors";
+import {
+  fimsHistoryToUndefinedSelector,
+  isFimsHistoryLoadingSelector
+} from "../store/selectors";
 
 export const FimsHistoryScreen = () => {
   const dispatch = useIODispatch();
-  const historyPot = useIOSelector(fimsHistoryPotSelector);
+  const isLoading = useIOSelector(isFimsHistoryLoadingSelector);
+  const consents = useIOSelector(fimsHistoryToUndefinedSelector);
 
   React.useEffect(() => {
-    dispatch(fimsHistoryGet.request({}));
+    dispatch(fimsHistoryGet.request({ isFirstRequest: true }));
   }, [dispatch]);
 
+  const fetchMore = React.useCallback(() => {
+    if (consents?.continuationToken) {
+      dispatch(
```

Comment:
Using the io-dev-api-server, try the following scenario:
- the first request returns a page of history, with a continuation token
- now, stop and start the dev server and change the request response so that it always returns an errore
- reach the bottom of the list

Does it go into an infinite loop of requests?

Reply:
good catch, and yes it does 😅 
I was thinking of adding an error toast in order to give the user a bit more of info on the matter whenever it happens, but I also think we should talk this out at least with design.
How about I open a task about this and address it in the near future?

Reply:
Yeah, let's put it in the backlog for now